pkg/bindinfo/utils.go

```diff
@@ -167,6 +170,109 @@ func readBindingsFromStorage(sPool util.DestroyableSessionPool, condition string
 	return
 }
 
+var (
+	// UpdateBindingUsageInfoBatchSize indicates the batch size when updating binding usage info to storage.
+	UpdateBindingUsageInfoBatchSize = 100
+	// MaxWriteInterval indicates the interval at which a write operation needs to be performed after a binding has not been read.
+	MaxWriteInterval = 6 * time.Hour
+)
+
+func updateBindingUsageInfoToStorage(sPool util.DestroyableSessionPool, bindings []*Binding) error {
```

Comment:
Do we have any test data that how long this would take for 10,000 and 100,000 bindings?

Reply:
Please add some comments of its performance like `around 1m for 10,000 bindings`. I'm afraid users would ask us this in the future, better to leave some comments for it in case other developers don't know its performance.

Reply:
I create 44679 bindings and remove the limit condition to make tidb always write the binding info.

```
[2025/09/22 19:11:41.210 +08:00] [INFO] [utils.go:185] ["update binding usage info to storage"] [category=sql-bind] [count=44679] [duration=26.241038459s]
```

Reply:
it spends 26.241038459s to update 44679 bindings